readme.md

```diff
@@ -599,6 +599,7 @@
 - [Alfred Workflows](https://github.com/alfred-workflows/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
 - [GitHub Actions](https://github.com/sdras/awesome-actions#readme) - Create tasks to automate your workflow and share them with others on GitHub.
+- [Orgs](https://github.com/beansource/awesome-orgs#readme) - GitHub Organizations.
```

Comment:
```suggestion
- [GitHub Organizations](https://github.com/beansource/awesome-orgs#readme) - GitHub Organizations.
```

Reply:
The description would be repetitive otherwise.

Reply:
true. In that case the description doesn't have sense.
Maybe that could be description:
```
Organizations are shared accounts where businesses and open-source projects can collaborate across many projects at once, with sophisticated security and administrative features.
```

Reply:
How about just

>Shared accounts where businesses and open-source projects can collaborate across many projects at once on GitHub.

Reply:
Same Goals

Reply:
Positivity